packages/sdk/src/lib/types/internalTypes.ts

```diff
@@ -6,6 +6,13 @@ export type IExecConsumer = {
   iexec: IExec;
 };
 
+export type IExecDebugConsumer = {
```

Comment:
```suggestion
export type IExecDebugClient = {
```
Ca me parle plus 🤷🏻‍♂️